_pages/platform/usage-metrics.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: article
+title:  "Usage Mestrics"
```

Comment:
Typo - I'm going to get some consensus on what we are calling this feature in product meeting today, I've talked a bit to other PM's last week and thinking something along the lines of "Usage Reports" but "Usage Metrics" works too - will let you know by EOD today. This doc looks great and thanks so much for doing it!